ocs_ci/utility/utils.py

```diff
@@ -1997,6 +1997,34 @@ def destroy_cluster(installer, cluster_path, log_level="DEBUG"):
         log.error(traceback.format_exc())
 
 
+def config_to_string(config):
+    """
+    Convert ConfigParser object to string in INI format.
+
+    Args:
+        config (obj): ConfigParser object
+
+    Returns:
+        str: Config in one string
+
+    """
+    string = ""
+    sections = [config.default_section] + config.sections()
+
+    first = True
```

Comment:
same

Reply:
What name do you propose? I's part of the algorithm and it should contain information if the iteration is first or not.

Reply:
could be `first_iteration` then